src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -57,9 +64,21 @@ static PSVersionInfo()
         {
             s_psVersionTable = new PSVersionHashTable(StringComparer.OrdinalIgnoreCase);
 
+            // Get '6.0.0-beta.7 Commits: 29 SHA: 52c6bfe1eae24dbaa1a162bffb3754ba3fdc1f4c'.
+            // The formatted string is displayed to users in '$PSVersionTable' and PowerShell banner.
```

Comment:
We shouldn't use the formatted string from the attribute directly. CommitId should be in the exact same format as before.

Reply:
Reverted.